src/Microsoft.DotNet.GenAPI/Shared/GenAPIApp.cs

```
@@ -148,4 +167,15 @@ public static string ReadHeaderFile(string? headerFile)
         }
         return defaultFileHeader;
     }
+
+    private static void AddReferenceToRuntimeLibraries(IAssemblySymbolLoader loader)
+    {
+        var corlibLocation = typeof(object).Assembly.Location;
+        var runtimeFolder = Path.GetDirectoryName(corlibLocation);
+
+        if (runtimeFolder != null)
```

Should we silently not add the the reference or should we throw in case the path is not available for some reason?

Btw the reason `Location` can be null is that assembly can be also loaded from memory (not from a file). And `GetDirectoryName` is obvious file system issues.

But maybe we want to know about this and throw?

---

Btw in C# you can do this:
```csharp
var runtimeFolder = Path.GetDirectoryName(corlibLocation) ?? throw new ArgumentNullException("RuntimeFolder", "Could not find path to a runtime folder");

loader.AddReferenceSearchDirectory(runtimeFolder);
```